yw_website/yw_website/apps/website/templates/includeable/workflow_list.html

```diff
@@ -26,37 +26,42 @@
 
         {% for workflow in workflow_list %}
             <li>
-                <div classs = "tags">
-                    Tags: {{ workflow.tags | join:", " }}
-                </div>
                 <div class = "title">
-                    {{ workflow.title|linebreaks }}
+                        {{ workflow.title|linebreaks }}
+                    
                 </div>
                 <div class = "last_modified"> 
                     {{ workflow.version_modified }}
                 </div>
 
-                <div id="graph{{ forloop.counter }}" class = workflow-image></div>
+                <!-- <div id="graph{{ forloop.counter }}" class = workflow-image></div> -->
 
-                <div class = "workflow-details" id="description">
+                
+                <div  class = "workflow-description" id="description">
```

Comment:
is there a reason this needs an id? this is also violating having id's be unique